source/content/index.md

```diff
@@ -1,7 +1,30 @@
 ---
 title: Vogelfrei
 ---
-Contents of Vogelfrei game:
+## What is Vogelfrei?
+
+>
+> *For some, exile is the only fate. To be **vogelfrei** is to be cast out, beyond the protection of kings and gods alike, free in the most terrible way: free to be hunted, betrayed, and forgotten.*
+> 
+> *Yet even outlaws have a chance to carve their own destiny. A knight may fall from grace and take up the life of a mercenary. A peasant may flee the fields and become a bandit lord. A scholar may seek forbidden knowledge, risking soul and sanity. Steel and wit are a person’s greatest allies, for dangers lurk in the wilds and in the depths of the unknown. In this place, you are either bound by the laws of men—or cast out, left to forge your own destiny.*
+
+**Vogelfrei** is an OSR tabletop role-playing game set in a world shaped by feudal and early Renaissance societies. Players create characters from diverse social backgrounds — peasants, academics, warriors, rangers and rogues — where social status heavily influences life in civilized settlements. However, in untamed lands and forgotten ruins, the established order begins to crumble. This game emphasizes exploration across vast wildernesses — forests, mountains, and wastelands — as well as deep dungeons and the remains of lost cities.
```

Comment:
First thing mentioned is status, which is nowhere near to be the core of the game

Reply:
We can specify that status only matters in towns/cities while game is set in uncivilized lands mostly?